framework/source/class/qx/ui/table/model/Filtered.js

```diff
@@ -105,6 +105,14 @@ qx.Class.define("qx.ui.table.model.Filtered",
     __applyingFilters : null,
     __filterTypes : null,
 
+    /**
+     * Function to get the full array of the filtered model
+     * @return {Array} the full array of model (with no changes)
+    */
+      getFullArray: function ()
+    {
+	return this.__fullArr;
```

Comment:
Please fix the indentation.

Reply:
Hi, we already fix the indentation. We will review it but at the moment the change help us a lot. Thanks!